src/main/resources/lib/repo/statreg/contacts.ts

```diff
@@ -9,5 +9,5 @@ export function fetchContacts() {
 
 export function getContactsFromRepo() {
   const node: StatRegNode | null = getStatRegNode(STATREG_REPO_CONTACTS_KEY)
-  return node ? node.content : null
+  return node ? (node.content) : null
```

Comment:
Får jeg spørre hvorfor du har parantes rundt `node.content` ?

Reply:
i think i was trying something with (node.content as <Type>), removed it later but forgot to remove the parenthesis. fixed.